bin/p2-launch/main.go

```diff
@@ -88,7 +88,8 @@ func main() {
 	podFactory := pods.NewFactory(*podRoot, types.NodeName(*nodeName), fetcher, "")
 	pod := podFactory.NewLegacyPod(manifest.ID())
 
-	err = pod.Install(manifest, auth.NopVerifier(), artifact.NewRegistry(*artifactRegistryURL, fetcher, osversion.DefaultDetector))
+	// TODO: config store/labeler? Otherwise, p2-launch ignores the config store, which doesn't seem ideal.
```

Comment:
hmmmmmmmmmmmmmmmmmmmmmm yeah we're really hampering `p2-launch` here because now the manifest isn't all the info you need to run a pod